maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java

```diff
@@ -112,11 +113,17 @@
         if (parent == null) {
             throw new MojoFailureException("No <parent> found");
         }
-        if (!parent.getDependencyConflictId().equals("org.jenkins-ci.plugins:plugin:pom")) {
+        if (!(parent.getDependencyConflictId().equals("org.jenkins-ci.plugins:plugin:pom") || parent.getDependencyConflictId().equals("org.jenkins-ci:jenkins:pom"))) {
             throw new MojoFailureException("Unexpected <parent> " + parent);
         }
-        if (new ComparableVersion(parent.getVersion()).compareTo(new ComparableVersion(MINIMUM_PARENT)) < 0) {
-            PomHelper.setProjectParentVersion(pom, MINIMUM_PARENT);
+        String minimum_parent;
+        if (parent.getDependencyConflictId().equals("org.jenkins-ci:jenkins:pom")) {
```

Comment:
Maybe useful to introduce a variable for the parent ID.

Reply:
Will do!